test/multiCallHelper.js

```diff
@@ -0,0 +1,114 @@
+/* global artifacts */
+
+const ethers = require("ethers");
+const chai = require("chai");
+const BN = require("bn.js");
+const bnChai = require("bn-chai");
+
+const { assert } = chai;
+chai.use(bnChai(BN));
+
+const Proxy = artifacts.require("Proxy");
+const BaseWallet = artifacts.require("BaseWallet");
+const Registry = artifacts.require("ModuleRegistry");
+const TransferStorage = artifacts.require("TransferStorage");
+const GuardianStorage = artifacts.require("GuardianStorage");
+const ArgentModule = artifacts.require("ArgentModule");
+const DappRegistry = artifacts.require("DappRegistry");
+const UniswapV2Router01 = artifacts.require("DummyUniV2Router");
+const MultiCallHelper = artifacts.require("MultiCallHelper");
+
+const { encodeTransaction, addTrustedContact } = require("../utils/utilities.js");
+
+const ZERO_BYTES = "0x";
+const ZERO_ADDRESS = ethers.constants.AddressZero;
+const SECURITY_PERIOD = 2;
+const SECURITY_WINDOW = 2;
+const LOCK_PERIOD = 4;
+const RECOVERY_PERIOD = 4;
+
+const MAX_UINT = (new BN(2)).pow(new BN(256)) - 1;
```

Comment:
Sorry but this is again casting to a `Number` with loss of precision. You need to use `(new BN(2)).pow(new BN(256)).sub(new BN(1))`

Reply:
For reference `sub(new BN(1))` has the shorthand `subn(1)` but approving this as-is for now